docs/articles/book/concepts/TestingConcepts.md

```diff
@@ -0,0 +1,88 @@
+# Testing Concepts
+
+Lots of concepts are thrown around when we talk about automated testing. We'll briefly describe some of these here and then we'll work on applying them in the rest of the series.
+
+## SUT / CUT (Situation/Class Under Test)
+
+You might hear the term `SUT` or `CUT` or see these variables in tests you come across. Typically they mean the situation or class under test.
```

Comment:
I don't think that it needs to, and I've commonly taught it as situation because it helps students get in the mindset of thinking what they're actually testing, in which case system may be entirely too broad or confusing.

Reply:
Well, https://glossary.istqb.org/en_US/search?term=SUT   and also https://acronyms.thefreedictionary.com/SUT.  I don't think it is wrong to use situation, but I would add System to the list there.  It makes it inline with what I think is a more common use of the acronym